scripts/centos/vmware.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -eux
```

Comment:
Hi @cheeseplus,

Changed it for Centos 7. Still need to test OVT with older versions. What do you mean by "abstract all of them"?